etc/docker/demo-base/Dockerfile

```diff
@@ -49,11 +49,16 @@ RUN apt-get update && apt-get -yq dist-upgrade \
     libsm6 \
     libxext-dev \
     libxrender1 \
-    openjdk-8-jdk \
+    software-properties-common \
     openssh-server \
     openssh-client \
+ && apt-add-repository 'deb http://security.debian.org/debian-security stretch/updates main' \
+ && apt-get update && apt-get install -yq --no-install-recommends openjdk-8-jdk \
  && rm -rf /var/lib/apt/lists/*
 
+RUN mkdir -p /usr/lib/java
```

Comment:
Per https://github.com/jupyter-server/enterprise_gateway/pull/1194/files#r1020484617 this can be removed.  It's actually a bug anyway (should be `/usr/lib/jvm/java`) - which is why this particular image is working...
```suggestion
```